.github/workflows/build.yml

```diff
@@ -19,13 +19,47 @@ jobs:
         run: cargo build --verbose
       - name: Run unit tests
         run: cargo test --verbose
-      - name: Verify if bindings change
-        uses: tj-actions/verify-changed-files@v11.1
-        id: verify-bindings
+  build-ios:
+    name: Build for iOS targets
+    runs-on: macos-latest
+    strategy:
+      matrix:
+        target: [ x86_64-apple-ios, aarch64-apple-ios, aarch64-apple-ios-sim ]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+      - name: Install rust
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          profile: minimal
+          override: true
+          target: ${{ matrix.target }}
+      - name: Build target
+        uses: actions-rs/cargo@v1
+        with:
+          use-cross: false
+          command: build
```

Comment:
Shouldn't we add `test` step to make sure that we test the code for target platforms?

Reply:
We generally can't because the binaries are being cross-compiled for other targets/architectures.